contributing.md

```diff
@@ -0,0 +1,153 @@
+|`index`|string|No| If a section landing page needs to be different from the `index.md` file in that directory, this field specifies the corresponding Markdown file name. | 
+
+#### Partials
+
+Partials allow you to reuse snippets of content in multiple places. All partials
+live in the `_partials` top-level directory. To make a new partial, create a new
+`.md` file in this directory. The filename must start with an underscore. Then import it into the target page and reference in the relevant place. See [Formatting examples](_formatting_examples.md).
+
+### Formatting
+
+In addition to all the [regular Markdown formatting][markdown-syntax], the following elements are available for Timescale docs:
+
+- Procedure blocks 
+- Highlight blocks
+- Tabs
+- Custom code blocks
+- Multi-tab code blocks
+- Tags
+
+See [Formatting examples](_formatting_examples.md) for how to use them. 
+
+### Variables
+
+Timescale documentation uses variables for its product names, features, and UI elements in Timescale Console with the following syntax: `$VARIABLE_NAME`. Variables do not work inside the following: 
+
+- Front matter on each page
+- HTML tables
+
+The main variables include the following:
+
+```text
+COMPANY = Timescale
+COMPANY_URL = https://www.timescale.com
+CLOUD_LONG = Timescale Cloud
+TIMESCALE_DB = TimescaleDB
+CONSOLE = Timescale Console
+CONSOLE_URL = https://console.cloud.timescale.com/
+SERVICE_LONG = Timescale Cloud service
+SERVICE_SHORT = service
+```
+
+The list of variables is an ever-evolving document. Rely on the help of our doc team to apply all relevant ones when reviewing your contribution. 
+
+### Links
+
+Links should be reference-style links where the link address is at the bottom of the page. 
+
+- Internal page links: internal links do not need to include the domain name `https://docs.timescale.com`. Use the `:currentVersion:` variable instead of `latest` in the URL.
+- External links: input external links as is. 
+
+See [Formatting examples](_formatting_examples.md) for details. 
+
+### Visuals
+
+When adding screenshots to the docs, aim for a full-screen view to provide better context. Attach the image to your issue or PR, and the doc team will upload and insert it for you.
```

Comment:
Full screen, but reduce the size of your browser so there is as little wasted space as possible.